src/main/java/seedu/taassist/model/util/SampleDataUtil.java

```diff
@@ -21,22 +21,22 @@ public static Student[] getSampleStudents() {
         return new Student[] {
             new Student(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends")),
+                getModuleClassSet("friends")),
```

Comment:
Perhaps consider changing the data to be more "module-like"? i.e. CS1231S etc. Albeit this is not important and can be noted down as an issue for later.

Reply:
^ I'll probably add this as another issue. If not bubbl3t3a has too much to change

Reply:
I have done the relevant changes to some parts of the code. But for now, there isn't a systematic scan to the whole project to fix all such cases, since there are multiple dummy data in the test cases too. We should focus this in another issue when we fix the test cases too.